templates/status/500.tmpl

```diff
@@ -9,6 +9,8 @@
 <head>
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>Internal Server Error - {{AppName}}</title>
+	{{if ThemeColorMetaTag}}<meta name="theme-color" content="{{ThemeColorMetaTag}}">{{end}}
+	<meta name="default-theme" content="{{DefaultTheme}}">
```

Comment:
Theme related code could be put into `base/stylesheets.tmpl` , the `base/stylesheets.tmpl` already handles theme related resources/attributes.

ps: I didn't find any reference for `<meta name="default-theme"`, I consider it as no-op and should be removed.

Reply:
The meta is from https://github.com/go-gitea/gitea/pull/13809, it is not generally recognized.

Reply:
I can move that stuff, but need to find a new filename then.

Reply:
> The meta is from #13809, it is not generally recognized.

I am afraid it was a misunderstanding from discord discussion ..... if it is useful or used by any screen reader, why there is no single result on internet? I have tried to search "default-theme" with some different keywords, I can't see any information for it.

Reply:
I think the `<meta name="default-theme" ` could be safely removed (or commented out), if any user is affected, collect the full information and write a comment for it.

Reply:
ps: even worse, this `<meta name="default-theme"` can't be right for users, because it's just the site's default theme, not the user's theme. From various perspectives, this tag couldn't be right.

Reply:
Yes, I think we could just remove it. It's incorrect and a random discord request seems a bit of a weak justification to have it.

Reply:
Removed it.